src/util/not-empty.ts

```diff
@@ -0,0 +1,7 @@
+const notEmpty = <TValue>(
```

Comment:
this is just super useful for Typscript to filter out potentially null items from your array